src/Pages/Signup/index.scss

```
@@ -0,0 +1,396 @@
+.signup {
+  min-width: 320px;
+  display: flex;
+  align-content: center;
+  justify-content: center;
+
+  .signupContainer {
+    width: 100%;
+    height: 100%;
+    padding: 50px 0;
+
+    .header {
+      max-width: 520px;
+      text-align: center;
+      margin: 0 auto;
+
+      .logo {
+        color: #252525;
+        display: inline-block;
+        width: 100px;
+        margin: 0 auto;
+        line-height: 1;
+        vertical-align: top;
+        font-size: 36px;
+      }
+    }
+    .main {
+      .mainWrap {
+        position: relative;
+        padding: 58px 69px;
+        border: 1px solid #e5e5e5;
+        border-radius: 2px;
+        max-width: 580px;
+        margin: 40px auto;
+
+        .mainTitle {
+          font-size: 24px;
+          line-height: 34px;
+          color: #252525;
+          font-weight: normal;
+          margin-bottom: 23px;
+        }
+
+        .mainForm {
+          .email {
+            .emailTitle {
```

```suggestion
            .emailTitle {
```
스타일이 없고 불필요한 경우 굳이 셀렉터로 명시해줄 필요 없습니다

---

수정하였습니다!